cogs/ocr.py

```diff
@@ -27,83 +28,47 @@ def run_easyocr() -> list:
     cv2.imwrite(IMAGE_GRAY, gray_image)
 
     # Binary Image with Thresholding
-    thresh, im_bw = cv2.threshold(gray_image, 165, 255, cv2.THRESH_BINARY)
+    _, im_bw = cv2.threshold(gray_image, 165, 255, cv2.THRESH_BINARY)
+    pixel_width = im_bw.shape[0]
     cv2.imwrite('image-final.png', im_bw)
 
     reader = easyocr.Reader(['en'])
     results = reader.readtext(IMAGE_RESIZED)
-
```

Comment:
the sorting should be it's own function. Moved it to the utils file, and called it instead.